code/uri/1002.rb

```diff
@@ -0,0 +1,5 @@
+pi_const= 3.14159
+r = gets.to_f
+a = pi_const*r*r
```

Comment:
É legal sempre separar os números dos operadores. Essa linha ficaria assim:

```suggestion
a = pi_const * r * r
```